portfolio/src/main/webapp/about.html

```diff
@@ -26,24 +27,59 @@
       </nav>
     </div>
 
-    <div id="content">
-      <h1 id="title">My Life</h1>
-      <p>Just to explain my background...</p>
-      <p>I am originally from Chicago! Well, a small suburb called Long Grove :p</p>
-      <img src='/images/longgrove.jpg'>
-      <figcaption>This is my hometown! Cozy</figcaption>
+    <div class="aboutimg1">
+      <div class="titlecontainer">
+        <span class="homepageHeader1">My Life</span>
+      </div>
+    </div>
+
+
+    <div class="aboutimg2">
+      <div class="abouttext">
+        <span class="border trans">
+          Long Grove
+        </span>
+      </div>
+    </div>
 
-      <p>I currently attend Stanford University! I study computer science. Beep boop. I've also
-          taken classes on philosophy, molecular gastronomy(read: foooood), and earth science!
+    <section class="section section-dark">
+      <p>
+        I'm from a small suburb of Chicago called Long Grove. It's a place that relies on minimal technology - you can even see people
+        riding on horses (sometimes)! Now that I'm in the tech field, my simple roots have shaped my sense of purpose
+        into something that lies between the intersection of complex innovation and community values. I hope to create new 
+        technologies and imbue traditional cultures with them.
       </p>
-      <p>Outside of class, I really enjoy playing volleyball, basketball, badminton, and spikeball! I also occasionally
-          find myself playing an unhealhty amount of League of Legends. Add me @gucci gan. When I'm not in the gym or the
-          computer lounge, you'll probably find me exploring beaches(Big Sur, Half Moon Bay, etc.) and consuming good eats.
-          Here are some pictures from the school year that describe me :D </p>
-      
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+    </section>
+
+    <div class="aboutimg3">
+      <div class="abouttext">
+        <span class="border trans" style="color:red">
+          Stanford
+        </span>
+      </div>
     </div>
+
+    <section class="section section-dark">
+      <p>
+        I currently attend Stanford University! I study computer science. Beep boop. I've also
+        taken classes on philosophy, molecular gastronomy(read: foooood), and earth science!
+      </p>
+    </section>
+
+    <div class="aboutimg4">
+      <div class="abouttext">
+        <span class="border trans" style="color:green">
```

Comment:
Try to avoid inline styling since it becomes unwieldy when debugging CSS definitions.

Reply:
what would be a better practice? if I want to change two texts' colors that are otherwise identical stylistically